### DISCUSSION TITLE
Part notes visible from build notes?

### DISCUSSION THREAD

**Discussion by StormTrooper** [inventree/InvenTree#5513] [Q&A]
This maybe a FR (Unless there is some easier way to do this?)

I have a part listed which is built from other stock items, and have the  required BOM. In the part notes I have a link to the KiCad interactive HTML BOM. 

When building a new part under the build orders I would like to be able to view the HTML BOM link from the notes. The note in the build note is blank so I'm guessing this is just  a note specific for the build. It would be handy to be able to view the part note and then load up the interactive BOM link. 

I now have to open the part and then view the note - not the end of the world, maybe I'm just being lazy :)

**Comment by matmair** [comment]
That is a FR, build notes are per build currently. FYI we are rewriting the whole UI #5212
